tests/split_test.py

```diff
@@ -1,26 +1,36 @@
-# from steps.split import process_splits
-# from pandas import DataFrame
+import pytest
+import os
+import pandas as pd
+from pandas import DataFrame
+from steps.split import process_splits
 
-# TODO(apurva-koti): uncomment these asap and update to match new split process fn state
-# def test_post_split_fn_returns_datasets_with_correct_spec():
-#     train = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]})
-#     validation = DataFrame({"a": [1], "b": [4], "c": [7]})
-#     test = DataFrame({"a": [2], "b": [6], "c": [9]})
-#     (train_processed, validation_processed, test_processed) = process_splits(
-#         train, validation, test
-#     )
-#     assert isinstance(train_processed, DataFrame)
-#     assert isinstance(validation_processed, DataFrame)
-#     assert isinstance(test_processed, DataFrame)
-#
-#
-# def test_post_split_fn_returns_non_empty_datasets():
-#     train = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]})
-#     validation = DataFrame({"a": [1], "b": [4], "c": [7]})
-#     test = DataFrame({"a": [2], "b": [6], "c": [9]})
-#     (train_processed, validation_processed, test_processed) = process_splits(
-#         train, validation, test
-#     )
-#     assert not train_processed.empty
-#     assert not validation_processed.empty
-#     assert not test_processed.empty
+
+@pytest.fixture
+def sample_data():
+    return pd.read_parquet(
+        os.path.join(os.path.dirname(__file__), "test_sample.parquet")
+    )
+
+
+def test_post_split_fn_returns_datasets_with_correct_spec(sample_data):
```

Comment:
Check we check if the number of rows or the feature set is exactly the same?

Reply:
will check if the sum of rows across all splits is 10. Not sure if we should test on the feature set, because the split step may result in added or dropped features - hence we probably don't want to hardcode knowledge of the split function itself.